src/components/TextInput/index.tsx

```diff
@@ -11,6 +11,7 @@ type TextInputProps = {
     required?: boolean;
     placeholder?: string;
     'data-test-id'?: string;
+    onChange?: React.ChangeEventHandler<HTMLInputElement>;
```

Comment:
I think `onChange` could be optional, no?

```suggestion
    onChange: React.ChangeEventHandler<HTMLInputElement>;
```

Reply:
@tyrro do you mean `onChange` should be mandatory instead? 😅  By removing `?`, it becomes `required` now. I set `onChange` as optional because I don't need to set it in the unit test. 🙏

Reply:
@manh-t my bad 🙈 true, it's right as it is right now! 😄